src/main/java/cz/idomatojde/entity/Offer.java

```diff
@@ -0,0 +1,138 @@
+package cz.idomatojde.entity;
+
+import com.sun.istack.NotNull;
+
+import javax.persistence.*;
```

Comment:
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad